docker/test/wheels/debian.Dockerfile

```diff
@@ -24,7 +24,8 @@ ENV VIRTUAL_ENV=/opt/venv
 RUN python${python_version} -m venv "$VIRTUAL_ENV"
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 RUN if [ -n "$preinstalled_modules" ]; then \
-        echo $preinstalled_modules | xargs python${python_version} -m pip install; \
+        python${python_version} -m pip uninstall $preinstalled_modules ;\
```

Comment:
These files are used only for validation. Hence, anything in these files is what a user would need to do. 
These lines really don't do anything otherwise that would be a bug with pip.